### PR TITLE
Region from file

### PR DESCRIPTION
**Pull request by kgrochow**
- Moved region mapping definition to wwwroot/data/RegionMapping.json.  Defining valid regions can now be set by editing this file.
- Change displayable variables to just be scalar variables (dropped enumerated ones for now).
- Info box displays csv variables greater than 99999 with commas.
- General cleanup of CsvCatalogItem and tableStyle in prep for abs integration

### CODE REVIEW

File: lib/Models/CsvCatalogItem.js [hunk 1]
```diff
@@ -192,6 +193,20 @@ defineProperties(CsvCatalogItem.prototype, {
...
             return CsvCatalogItem.defaultPropertiesForSharing;
         }
+    },
+
+    /**
+     * Gets or sets the tableStyle object
+     * TODO: add definition for subfields
+     * @type {Object}
+     */
+    tableStyle :  {
```
Comment (kring) [inline lib/Models/CsvCatalogItem.js]: This isn't used because the `tableStyle` defined in the constructor (with the `knockout.track` line) overrides it.

**Comment by kring** [comment]
Looks good other than the one comment.

File: lib/Models/CsvCatalogItem.js (hunk 1 above)
Comment (kgrochow) [inline lib/Models/CsvCatalogItem.js]: It actually does get used and it's the only way I could get tableStyle to get handled properly across load, setting, and sharing.

Comment (kgrochow) [inline lib/Models/CsvCatalogItem.js]: Well it does get hit on loading.  I'll see if it works without it.

Comment (kgrochow) [inline lib/Models/CsvCatalogItem.js]: Yep, share doesn't work properly without it.

**Comment by kgrochow** [comment]
I moved the getter/setter and removed tableStyle from knockout.track.

**Pull request merged**